common-crafting.lic

```diff
@@ -346,5 +371,23 @@ module DRCC
       end
     end
   end
-    
+
+  def check_for_existing_sigil?(sigils, stock_number, quantity, bag, belt, info)
```

Comment:
Are you sending this a list of sigils?

Reply:
It should only receive one at a time by each or called with the enchant_name, etc, directly. See the updated `craft` script. `Workorders` is forth coming too. All working as I expect.

Reply:
If it's just checking one sigil, 'sigils' shouldn't be plural.